helix-term/src/ui/editor.rs

```diff
@@ -110,26 +110,26 @@ impl EditorView {
                     .and_then(|source| source.path.as_ref())
                     == doc.path()
             {
-                let line = frame.line - 1; // convert to 0-indexing
-                let style = theme.get("ui.highlight");
+                let line = frame.line;
+                let style = theme.get("ui.highlight.currentline");
```

Comment:
The original name was actually picked for DAP. We could maybe use `current` rather than `currentline`

Reply:
I was not aware of that. Maybe we could rename them entirely to `ui.debug.breakpoint` and `ui.debug.current`? Feels a bit more natural.

Reply:
The reason we would want to keep the `highlight` prefix here is that helix falls back to previous scope when a theme doesn't define a scope.

So if  a theme doesn't definine `ui.highlight.current` it would fallback to `ui.highlight` (which basically all themes definine already and is also the way I would expect the fallback to work).
However I agree that the name could maybe give away that it's actually a dab specific color (so it doesn't get reused again a bunch of times like `ui.highlight` did). Maybe we could name it `ui.highlight.currentframe`?

Reply:
I could also just go and rename `ui.highlight` to `ui.debug` for each theme, as it is a trivial change. Since highlight is not used for anything other than this and not mentioned in the docs (at least not in `book/src/themes.md`) it should be fine to give it a more meaningful name. I would also add a new key for the triangle character which @archseer requested in the issue.

Reply:
`ui.highlight` is also used for the line in the picker that is currently selected.

Reply:
ah I see, missed that. maybe it makes sense to separate them then? have `ui.highlight` and `ui.debug`. I'll have to update all themes, but it's not such a large number, and for the most part it's a trivial change.